CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 [Unreleased]: https://github.com/atomist/rug/compare/1.0.0-m.3...HEAD
 
+### Changed
+
+-   **BREAKING** Removed `label` from `Presentable` instruction
```

Comment:
We could.  I'm just thinking if label usage didn't propagate yet in Directed and Response message this would make for cleaner API - I'd like to migrate lifecycle messages and get rid of LabeledPresentable eventually.  I don't think anybody is using actions in Directed/Response messages yet since we just rolled that out.